addons/game.controller.genesis.4wayplay/addon.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="game.controller.genesis.4wayplay"
+        name="Genesis 4 Way Play"
+        version="1.0.0"
+        provider-name="Team Kodi">
+    <extension point="kodi.game.controller" library="resources/layout.xml"/>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <assets>
+            <icon>resources/icon.png</icon>
+        </assets>
+        <summary lang="en_GB">Genesis 4 Way Play</summary>
+        <description lang="en_GB">The 4 Way Play is a peripheral for the Sega Genesis, released in 1993 by Electronic Arts. It allows for up to four Genesis controllers to be connected. It was mainly supported by games released by EA itself.</description>
```

Comment:
To avoid the "wall of text" effect I try to limit descriptions to 2 short-ish sentences now. We can probably drop the middle sentence; the picture shows 4 controller ports.